src/core/pool_func.cpp

```diff
@@ -12,6 +12,10 @@
 #include "../stdafx.h"
 #include "pool_type.hpp"
 
+#include "../openttd.h"
+#include "../string_func.h"
+#include "../saveload/saveload.h"
```

Comment:
fair bit of overhead adding 3 header includes. Perhaps some forward declarations might be easier on the compiler?